students/minji/requirements.txt

```
@@ -0,0 +1,5 @@
+Django==3.2.4
+django-cors-headers==3.7.0
+mysqlclient==2.0.3
+    
+PyMySQL==1.0.2
```

```suggestion
Django==3.2.4
django-cors-headers==3.7.0
mysqlclient==2.0.3
PyMySQL==1.0.2
```
해당 파일에서 불필요한 공백은 지워주세요~!